### ISSUE TITLE
Missing moment.unix()

### ISSUE THREAD

**Issue by envil** [date-fns/date-fns#830]
I tried to find the function similar to `moment.unix()` in date-fns but could find. Is it me couldn't find it or it's indeed not exist?

**Comment by leshakoss** [comment]
The best way to do it right now is to use `getTime` function which returns the milliseconds timestamp.

```js
Math.floor(getTime(new Date()) / 1000)
```

**Comment by envil** [comment]
@leshakoss what I meant is to create a new Date object from the unix timestamp. I'm currently doing it like `const date = new Date(epoch * 1000)`

**Comment by leshakoss** [comment]
Oh, my bad. Every function in `date-fns` does accept a milliseconds timestamp instead of any date argument, so you can use them the same way that you're currently doing it: `getISOWeeksInYear(epoch * 1000)`. Also [`parse` from v1](
https://date-fns.org/v1.29.0/docs/parse) and [`toDate` from v2](
https://date-fns.org/v2.0.0-alpha.16/docs/toDate) could be used to convert ms timestamps to a date, and of course, using `new Date` constructor is a perfectly valid way to do it.

**Comment by envil** [comment]
Yep, my situation is I have to initiate the date from epoch time quite a lot in my application, so having the `* 1000` magic number everywhere is quite irritating to me. I could write my own date util to create the date like `fromUnix(epoch)` that can return the date, but then what's the point to use a some library like date-fns. Also not many people know how to create the date from the epoch time, so this would be quite helpful I think.

**Comment by Kingwl** [comment]
👍 helpful to me,
i prefer to explicit convert rather than `* 1000`

**Comment by kossnocorp** [comment]
That's a good idea and easy to implement. My API suggestion is two functions: `fromUnix()` and `toUnix()`. 

I'm strictly against being "smart" and trying to guess if it's JS time or Unix time because that would cause unpleasant bugs. Also given that you receive the time in seconds, it's vital to know that JS has time in milliseconds because likely you will send the time back to the back-end.

Also, I recommend using ISO-8601 strings for cross-language communications because it does not only save you from this kind of problems but also preserves the original time zone.

**Comment by Kingwl** [comment]
I have already sent a pr for `toUnix` but named `getUnixTime `
https://github.com/date-fns/date-fns/pull/870

**Comment by kossnocorp** [comment]
@Kingwl oh wow, I did miss it 🤦‍♂️ Thank you a lot, I'll take a look.

**Comment by xkizer** [comment]
PR for this in #937

**Comment by kossnocorp** [comment]
`fromUnixTime` was released as `date-fns@2.0.0-alpha.24`: https://gist.github.com/kossnocorp/a307a464760b405bb78ef5020a4ab136#v200-alpha24
